io.github.halfmexican.Mingle.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "io.github.halfmexican.Mingle",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "mingle",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/vala/bin",
+        "prepend-ld-library-path" : "/usr/lib/sdk/vala/lib"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+          "name": "blueprint-compiler",
+          "buildsystem": "meson",
+          "cleanup": ["*"],
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+              "tag": "v0.12.0"
+            }
+          ]
+        },
+        {
+            "name" : "mingle",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/halfmexican/mingle.git",
+                    "commit": "fdaacaecea3131725643680250006ed58d3d7c1f",
+                    "tag":  "v0.10"
```

Comment:
this tag doesn't exist.

Reply:
I'm so sorry for my incompetence.
it should exist [now](https://github.com/halfmexican/mingle/releases/tag/v0.10)